src/main/java/sirius/web/templates/pdf/handlers/BarcodePdfReplaceHandler.java

```diff
@@ -73,65 +69,39 @@ public FSImage resolveUri(String uri, UserAgentCallback userAgentCallback, int c
         return fsImage;
     }
 
-    private int calculateBarcodeScaleFactor(int cssWidth, int cssHeight, Image awtImage) {
-        return (int) Math.max(Math.ceil(cssWidth / (float) awtImage.getWidth(null)),
-                              Math.ceil(cssHeight / (float) awtImage.getHeight(null)));
-    }
+    private Image getBarcodeImage(String barcodeType, String content, int width, int height) throws WriterException {
```

Comment:
Shouldnt we name those methods something other than `get...` as they are not getters but do more work? Something like `generateBarcodeImage` for example